tests/sync/test_rest_api.py

```
@@ -224,29 +240,56 @@
+            pass
+        except Exception as e:
+            print(f"Entity extraction WebSocket error: {e}")
             pass
         finally:
-            await seb_communicator.disconnect()
-        await asyncio.sleep(5)  # Give a second for the fire and forget pubmed to run
+            if seb_communicator:
+                try:
+                    await seb_communicator.disconnect()
+                except:
+                    pass
```

Ruff (0.11.9)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)